sys-cluster/pacemaker/pacemaker-2.1.2.ebuild

```diff
@@ -54,19 +55,18 @@ src_configure() {
 		--with-ais
 		$(use_with acl)
 		$(use_with smtp esmtp)
-		$(use_with snmp)
 	)
-
 	econf "${myconf[@]}"
 }
 
 src_install() {
 	default
 	python_optimize
 
-	# remove provided initd file as we need support for OpenRC
-	rm -r "${ED}/etc/init.d" || die "Failed to remove old initd"
-	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	# Remove provided initd file as we need support for OpenRC
```

Comment:
You didn't address my comment yet btw

Reply:
Which one?

Reply:
https://github.com/gentoo/gentoo/pull/23636#discussion_r777686489